crates/fuel-gas-price-algorithm/src/v1.rs

```diff
@@ -59,6 +63,27 @@ impl AlgorithmV1 {
     }
 }
 
+pub type Height = u32;
+pub type Bytes = u64;
+
+pub trait UnrecordedBlocks {
+    fn insert(&mut self, height: Height, bytes: Bytes) -> Result<(), String>;
+
+    fn remove(&mut self, height: &Height) -> Result<Option<Bytes>, String>;
+}
+
+impl UnrecordedBlocks for BTreeMap<Height, Bytes> {
```

Comment:
It's mainly a holdover from before. It used to be that we'd iterate over them in order, but that requirement was removed. It's still likely that they'll be accessed in order, but `HashMap` probably works just fine here.

Reply:
Fair enough :+1: I also think `BTreeMap` feels better since the data is ordered, but just looking at the access pattern I can't really motivate it :sweat_smile: